HW8/marian_hrychak/task_1/utils/__init__.py

```
@@ -0,0 +1,4 @@
+from .formatter import *
```

Ruff (0.8.2)

`from .formatter import *` used; unable to detect undefined names

(F403)